plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2063,234 @@ def coupling_parameter(
+
+        These equations yield the exact formulas:
+
+            momentum loss - :math:`ν_{s}^{α\backslashβ}=\left(1+\frac{m_{α}}{m_{β}}\right)ψ\left(x^{α\backslashβ}\right)ν_{0}^{α\backslashβ}`
+
+            transverse diffusion - :math:`ν_{⊥}^{α\backslashβ}=2\left[\left(1-\frac{1}{2x^{α\backslashβ}}\right)ψ\left(x^{α\backslashβ}\right)\ +ψ'\left(x^{α\backslashβ}\right)\right]ν_{0}^{α\backslashβ}`
+
+            parallel diffusion - :math:`ν_{||}^{α\backslashβ}=\left[\frac{ψ\left(x^{α\backslashβ}\right)}{x^{α\backslashβ}}\right]ν_{0}^{α\backslashβ}`
+
+            energy loss - :math:`ν_{ε}^{α\backslashβ}=2\left[\left(\frac{m_{α}}{m_{β}}\right)ψ\left(x^{α\backslashβ}\right)-ψ'\left(x^{α\backslashβ}\right)\right]ν_{0}^{α\backslashβ}`
+
+        where,
+
+            :math:`ν_{0}^{α\backslashβ}=\frac{4\pi e_{α}^{2}e_{β}^{2}λ_{αβ}n_{β}}{m_{α}^{2}v_{α}^{3}}`,
+
+            :math:`x^{α\backslashβ}=\frac{m_{β}v_{α}^{2}}{2kT_{β}}`,
+
+            :math:`ψ\left(x\right)=\frac{2}{\sqrt{\pi}}\int_{0}^{x}t^{\frac{1}{2}}e^{-t}dt`, and
+
+            :math:`ψ'\left(x\right)=\frac{dψ}{dx}`
+
+        For limiting cases we encourage the curious reader to refer to p. 31 of :cite:t:`nrlformulary:2019`
+
+        This function uses CGS units under the hood to coincide with our references, but passing MKS units is acceptable.
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies.momentum_loss
+        <Quantity 1.83701432e+11 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+        """
+
+        # Note: This function uses CGS units internally to coincide with our references.
+        # Input is taken in MKS units and then converted as necessary. Output is in MKS units.
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle).to(u.cm / u.s)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+        mass_ratio = test_particle.mass / field_particle.mass
+
+        self.momentum_loss = (
```

Comment:
So the way you have it written now, it will re-compute the properties phi, phi_prime, and Lorentz_collision_frequency every time they are called. You could prevent this two ways: 

1) Use the [cached_property package](https://pypi.org/project/cached-property/) so that the value is stored after the first execution and then just returned. 
2) Do something like you were doing before with a variable, e.g. 

```python
phi = self.phi

slowing_down = .... * phi
transverse_diffusion = .... * phi
...
```

My preference would be towards the cached property option, since you are exposing `Lorentz_collision_frequency` and making it cached would mean that users of that would also get this benefit. You could also make `x` cached too for that reason.